source/MRViewer/MRSurfaceManipulationWidget.h

```diff
@@ -0,0 +1,62 @@
+#pragma once
+#include "MRViewerFwd.h"
+#include "MRViewerEventsListener.h"
+#include "MRMesh/MRMeshFwd.h"
+#include "MRViewer.h"
+#include "MRMesh/MRChangeMeshAction.h"
+
+namespace MR
+{
+
+class MRVIEWER_CLASS SurfaceManipulationWidget
```

Comment:
please add comments to this class

Reply:
fixed

Reply:
Can you also add comments to all pulbic functions, how to use them and what they do